examples/sender-constraints-get-200-mqtt.json

```diff
@@ -12,6 +12,18 @@
       "fixedTopicName2"
     ]
   },
+  "broker_protocol": {
```

Comment:
There's no point in either of these ones - the empty object has identical behaviour?